openshift_tools/monitoring/hawk_sender.py

```diff
@@ -90,7 +90,9 @@ def _get_default_hawk_connection(self):
 
         return hawk_connection
 
-    def add_metric(self, metrics, host=None, synthetic=False):
+    # key_tags is not changed inside method. disabling warning
+    # pylint: disable=dangerous-default-value
+    def add_metric(self, metrics, host=None, synthetic=False, key_tags={}):
```

Comment:
That's true that it's not changing right now, but it could change in a future PR. That future dev might not know about python's weird default value behavior and so won't change it then. That would cause weird, unexpected behavior.

We try to never disable pylint checks, especially when the fix is so easy.

Consider changing this to:
```python
     def add_metric(self, metrics, host=None, synthetic=False, key_tags=None):
￼          """ create unique metric from key value pair """
￼  
          if not key_tags:
              key_tags = {}

￼          if synthetic and not host:
```